mpt_extension_sdk/mpt_http/mpt.py

```diff
@@ -237,6 +242,40 @@ def get_product_onetime_items_by_ids(mpt_client, product_id, item_ids):
     return _paginated(mpt_client, url)
 
 
+@wrap_mpt_http_error
+def get_product_items_by_period(
```

Comment:
@svazquezco Su, could you please take a look at this SDK function? I remember that in your PRs to adobe there were some `get_onetime_items` sku that you were using. Do you think this SDK function can be reused?

It is for Su, nothing related to this PR :-)

Reply:
For sure! Right now we are using get_one_time_skus method to handle Assets and Subscriptions. 

<img width="606" height="357" alt="Image" src="https://github.com/user-attachments/assets/e754c12f-f7a4-4919-9d23-5e7d324a2ad6" />

They call -> [get_product_onetime_items_by_ids](https://github.com/softwareone-platform/mpt-extension-sdk/blob/main/mpt_extension_sdk/mpt_http/mpt.py#L265)

The main difference between them is the item condition but I'll check if we can use this method and remove the other one.